coalib/misc/Exceptions.py

```diff
@@ -20,12 +18,21 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        log_exception(log_message, exception)
         exitcode = 13
     elif isinstance(exception, BaseException):
-        log_printer.log_exception(Constants.CRASH_MESSAGE, exception)
+        log_exception(Constants.CRASH_MESSAGE, exception)
         exitcode = 255
     else:
         exitcode = 0
 
     return exitcode
+
+
+def log_exception(message, exception, log_level=LOG_LEVEL.ERROR):
```

Comment:
why not `log_level=logging.ERROR`?

Reply:
hm nvm, the log-levels are the same in `LOG_LEVEL` as in `logging`. We will unify this once we remove the deprecated features :+1: